postgres-appliance/scripts/configure_spilo.py

```diff
@@ -563,7 +562,7 @@ def get_placeholders(provider):
 
     placeholders['USE_WALE'] = any(placeholders.get(n) for n in AUTO_ENABLE_WALG_RESTORE +
                                    ('WAL_SWIFT_BUCKET', 'WALE_SWIFT_PREFIX', 'WAL_GCS_BUCKET',
-                                    'WAL_GS_BUCKET', 'WALE_GS_PREFIX', 'WALG_GS_PREFIX'))
+                                    'WAL_GS_BUCKET', 'WALE_GS_PREFIX', 'WALG_GS_PREFIX', 'WALG_AZ_PREFIX'))
```

Comment:
This change is not required, `AUTO_ENABLE_WALG_RESTORE` already contains `WALG_AZ_PREFIX`.